plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-roles.yaml

```diff
@@ -1508,6 +1508,22 @@ items:
     - create
     - patch
     - update
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    annotations:
+      rbac.authorization.kubernetes.io/autoupdate: "true"
+    labels:
+      kubernetes.io/bootstrapping: rbac-defaults
+    name: system:controller:staleness-prober
+  rules:
+  - apiGroups:
+    - ""
+    resources:
+    - pods
+    verbs:
+    - list
+    - watch
```

Comment:
Do we need "watch"?

Reply:
Don't think so, removed for least privilege.

Reply:
Looks like we might need it `TestControllerRoleVerbsConsistency` fails after removal.

Reply:
What is the test validating?